lib/prototyp/bigint.go

```diff
@@ -228,29 +246,58 @@ func (b *BigInt) UnmarshalBinary(buff []byte) error {
 
 func ParseNumberString(s string, base int) (int64, bool) {
 	var ns strings.Builder
-	if base == 10 {
+	switch base {
+	case 2:
+		for _, char := range s {
+			// 0-9 || B || b
+			if (char >= 48 && char <= 57) || char == 45 || char == 66 || char == 98 {
```

Comment:
Is there a reason `char == 45` is acceptable here but not in the other cases? We should update the comment if there's a specific reason.

Reply:
good catch, here is fix: https://github.com/0xsequence/go-sequence/pull/166